dev-packages/e2e-tests/test-applications/nestjs-distributed-tracing/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
```

Comment:
out of scope for this PR but idea for once #12920 is ready: In one of these apps we could set `"moduleResolution": "Node16"` to ensure our "workaround" from yesterday works in both, default and sub-path-export-compatible TS configs.